### PR TITLE
Update AWS next-steps.md

### PR DESCRIPTION
**Pull request by aaronkao**

### CODE REVIEW

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 1]
```diff
@@ -14,37 +14,34 @@ menu:
 aliases: ["/docs/quickstart/aws/next-steps/"]
 ---
 
-You've seen how to quickly get started using AWS with Pulumi.
+Congrats! You have deployed your first project on AWS with Pulumi. Here are the next steps:
 
-From here, you can dive deeper with additional AWS tutorials:
+## Learn Pulumi
+You can go through our Learn tutorials to have a comprehensive walkthrough of the key Pulumi concepts taught through the context of a real-life application.
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:20]:
```suggestion
## Learn Pulumi

You can go through our Learn tutorials to have a comprehensive walkthrough of the key Pulumi concepts taught through the context of a real-life application.
```

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 2]
```diff
@@ -14,37 +14,34 @@ menu:
...
 
-* [Containers on ECS "Fargate"]({{< relref "/registry/packages/aws/how-to-guides/ecs-fargate" >}}): Deploy containers to Amazon
-* [EC2 Linux WebServer VM]({{< relref "/registry/packages/aws/how-to-guides/ec2-webserver" >}}): Create an EC2 Linux Web Server virtual machine
-* [Serverless REST API Gateways using Lambda]({{< relref "/registry/packages/aws/how-to-guides/rest-api" >}}): Create simple RESTful web server using AWS Lambdas
-* [Serve a Static Website from S3]({{< relref "/registry/packages/aws/how-to-guides/s3-website" >}}): Serve a static website out of content in an S3 bucket
-* [Serverless + Containers + Infrastructure]({{< relref "/registry/packages/aws/how-to-guides/video-thumbnailer" >}}): Deploy a complete  application using a combination of buckets, serverless functions and containers.
+<div class="flex justify-left py-6">
+    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/learn/pulumi-fundamentals/" >}}">Learn Pulumi Fundamentals</a>
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:23]: It's better to only have one button of a primary color per page (one primary CTA) to help the user with other buttons being in secondary colors/secondary button types. Did the layout go through UX?

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 3]
```diff
@@ -14,37 +14,34 @@ menu:
...
+</div>
 
-In addition to the tutorials, several interesting examples are available complete with instructions:
+## How-to Guides
+You can go through our How-To Guides if there is a specific application stack or architecture that you wanted to deploy. These guides come in all the Pulumi languages and cover many common architectures such as [static websites]({{< relref "/registry/packages/aws-native/how-to-guides/aws-native-ts-s3-folder/" >}}), [EC2 virtual machines]({{< relref "/registry/packages/aws/how-to-guides/ec2-webserver" >}}), [EKS clusters]({{< relref "/registry/packages/aws/how-to-guides/aws-ts-eks/" >}}), [Fargate containers]({{< relref "/registry/packages/aws/how-to-guides/ecs-fargate" >}}), and [serverless applications]({{< relref "/registry/packages/aws/how-to-guides/rest-api" >}}).
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:27]:
```suggestion
## How-to Guides

You can go through our How-To Guides if there is a specific application stack or architecture that you want to deploy. These guides come in all the Pulumi languages and cover many common architectures such as [static websites]({{< relref "/registry/packages/aws-native/how-to-guides/aws-native-ts-s3-folder" >}}), [EC2 virtual machines]({{< relref "/registry/packages/aws/how-to-guides/ec2-webserver" >}}), [EKS clusters]({{< relref "/registry/packages/aws/how-to-guides/aws-ts-eks" >}}), [Fargate containers]({{< relref "/registry/packages/aws/how-to-guides/ecs-fargate" >}}), and [serverless applications]({{< relref "/registry/packages/aws/how-to-guides/rest-api" >}}). 
```

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 4]
```diff
@@ -14,37 +14,34 @@ menu:
...
-* [Ruby on Rails in EC2](https://github.com/pulumi/examples/tree/master/aws-ts-ruby-on-rails): Run a Ruby on Rails
-    WebServer using EC2 instances
-* [SQS-Triggered Lambdas](https://github.com/pulumi/examples/tree/master/aws-js-sqs-slack): Post to Slack using a Lambda
-    anytime a SQS message arrives
-* [Static Website in CloudFront and S3](https://github.com/pulumi/examples/tree/master/aws-ts-static-website): Create a
-    static website serving content out of S3, fronted by a CloudFront CDN
-* [Provision an Elastic Kubernetes Service Cluster](https://github.com/pulumi/examples/tree/master/aws-ts-eks): Stand up
-    a managed EKS cluster
+<div class="flex justify-left py-6">
+    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/registry/packages/aws/how-to-guides" >}}">Explore How-to Guides</a>
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:30]: Same note about buttons

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 5]
```diff
@@ -14,37 +14,34 @@ menu:
...
+</div>
 
-## Introducing Pulumi Crosswalk for AWS
+## How Pulumi Works
+You can read through our documentation that explains how Pulumi works from its architecture to key concepts, including [stacks]({{< relref "/docs/intro/concepts/stack" >}}), [state]({{< relref "/docs/intro/concepts/state" >}}), and [secrets]({{< relref "/docs/intro/concepts/secrets" >}}).
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:34]:
```suggestion
## How Pulumi Works

You can read through our documentation that explains how Pulumi works from its architecture to key concepts, including [stacks]({{< relref "/docs/intro/concepts/stack" >}}), [state]({{< relref "/docs/intro/concepts/state" >}}), and [secrets]({{< relref "/docs/intro/concepts/secrets" >}}).
```

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 6]
```diff
@@ -14,37 +14,34 @@ menu:
...
 
-<a href="{{< relref "/docs/guides/crosswalk/aws" >}}">
-    <img src="/images/docs/reference/crosswalk/aws/logo.svg" width="150" align="right" style="margin-left: 16px">
-</a>
+<div class="flex justify-left py-6">
+    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/docs/intro/concepts/" >}}">Read Documentation</a>
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:37]: Same note

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 7]
```diff
@@ -14,37 +14,34 @@ menu:
...
+</div>
 
-Use Pulumi Crosswalk for AWS to easily use the best of what AWS has to offer, with
-well-architected best practices, for the entire AWS cloud. Go to production
-with containers, Kubernetes, and serverless applications.
+## Blog Posts
+You can read through the latest blog posts about AWS that discuss everything from new AWS features and products supported by Pulumi to technical AWS architecture best practices.
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:41]:
```suggestion
## Blog Posts

You can read through the latest blog posts about AWS that discuss everything from new AWS features and products supported by Pulumi to technical AWS architecture best practices. 
```

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 8]
```diff
@@ -14,37 +14,34 @@ menu:
...
 
-Use Pulumi Crosswalk for AWS to easily use the best of what AWS has to offer, with
-well-architected best practices, for the entire AWS cloud. Go to production
-with containers, Kubernetes, and serverless applications.
+## Blog Posts
+You can read through the latest blog posts about AWS that discuss everything from new AWS features and products supported by Pulumi to technical AWS architecture best practices. 
 
-<a href="{{< relref "/docs/guides/crosswalk/aws" >}}">Get Started with Crosswalk for AWS Now</a>
+<div class="flex justify-left py-6">
+    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/blog/tag/aws/" >}}">Read the Pulumi Blog</a>
```
Comment (nimbinatus) [inline themes/default/content/docs/get-started/aws/next-steps.md:44]: Same note about the buttons

```suggestion
    <a class="btn btn-lg mx-1 my-1" href="{{< relref "/blog/tag/aws" >}}">Read the Pulumi Blog</a>
```

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Some notes; the links are broken because of the trailing slashes.

File: themes/default/content/docs/get-started/aws/next-steps.md [hunk 9]
```diff
@@ -14,37 +14,38 @@ menu:
 aliases: ["/docs/quickstart/aws/next-steps/"]
 ---
 
-You've seen how to quickly get started using AWS with Pulumi.
+Congrats! You have deployed your first project on AWS with Pulumi. Here are the next steps:
```
Comment (aaronkao) [inline themes/default/content/docs/get-started/aws/next-steps.md:17]: Should probably be "Here are the next steps depending on your learning style:"

**Pull request closed without merging**